packages/openactive-broker-microservice/app.js

```diff
@@ -987,6 +987,7 @@ app.post('/test-interface/datasets/:testDatasetIdentifier/opportunities', functi
     logError(`Random Bookable Opportunity from seller ${sellerId} for ${criteriaName} within ${bookingFlow} (${opportunityType}) call failed: No matching opportunities found`);
     res.status(404).json({
       error: `Opportunity of type '${opportunityType}' that has an \`organizer\`/\`provider\` with \`@id\` '${sellerId}', that also matched '${criteriaName}' within '${bookingFlow}', was not found within the feeds.`,
+      type: 'OpportunityNotFound',
```

Comment:
So that this 404 is not confused with other types of 404s when determining if there are missing opportunities